Microsoft.DotNet.Interactive/KernelBaseNugetExtensions.cs

```diff
@@ -0,0 +1,250 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.CommandLine.Parsing;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Html;
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.Events;
+using Microsoft.DotNet.Interactive.Formatting;
+using static Microsoft.DotNet.Interactive.Formatting.PocketViewTags;
+
+namespace Microsoft.DotNet.Interactive
+{
+    public static class KernelBaseNugetExtensions
+    {
+        public static T UseNugetDirective<T>(this T kernel) where T: KernelBase
```

Comment:
This won't make sense in the context of other kernels. I'd suggest we extract an interface and let people opt into this functionality.